tests/CommandsTest.php

```diff
@@ -76,6 +76,11 @@
 });
 
 test('migrate command loads schema state', function () {
+    if (! isVersion8()) {
```

Comment:
These changes can likewise be reverted. When exactly were they added @abrardev99?

In this commit? https://github.com/archtechx/tenancy/commit/f165fc58ba9b62b76798fdf0f9330319bceb7afd

Was that to support lower versions before we decided to send this to `4.x` (which will support fewer versions)?

You can probably just revert that one commit instead of making these changes manually.

Reply:
> When exactly were they added @abrardev99?
> In this commit? https://github.com/archtechx/tenancy/commit/f165fc58ba9b62b76798fdf0f9330319bceb7afd

Yes. 

> Was that to support lower versions before we decided to send this to 4.x

Yes. [Here you said](https://github.com/archtechx/tenancy/pull/793#issuecomment-1034026507) support lower versions. The master branch got some changes that need `PHP 8.1`. So we [decided](https://github.com/archtechx/tenancy/pull/793#issuecomment-1205622528) to support only `L8` and `L9` on `PHP 8.1`.